airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -50,6 +50,7 @@ class Meta:
         model = DagModel
 
     dag_id = auto_field(dump_only=True)
+    dag_display_name = fields.String(attribute="dag_display_name")
```

Comment:
Should this and `task_display_name` on TI also have `dump_only`? I don’t think it makes sense to be able to modify them.